content/docs/command-reference/params/index.md

```diff
@@ -58,10 +58,6 @@ as the tree path to find those values. Supported types are: string, integer,
 float, and arrays (groups of params). Note that DVC does not ascribe any
 specific meaning to these values.
 
-> YAML 1.2 stores very large and very small numbers in scientific notation, but
-> the popular PyYAML library uses an older version of the format. To avoid
-> introducing subtle bugs, the ruamel.yaml library should be used instead.
```

Comment:
This note is quite interesting (thanks @bobertlo) but doesn't seem necessary. This is a [general explanation about params](http://localhost:8000/doc/command-reference/params#description) (which should be extracted to a user guide eventually) and DVC in general is language-agnostic, so this very Python-specific note isn't required IMO.

Reply:
I see your point and will leave it up to you, but I think limiting this info to the examples defeats the purpose of the PR by @bobertlo. We aim to be language agnostic and this note is not really about DVC, but IMO we should err on the side of including warning about common issues that users face even if they are not strictly necessary.

Reply:
For context, I posted this because the mentioned subtle bug breaks user code by not parsing the scientific notation generated by experiments. After reading the error messages, the top google result for 'dvc params scientific notation' lead me an open issue from a year ago discussing whether or not to fix the bug. So this blurb is specifically for SEO and user experience.

Reply:
I think in any case we should find the note a better place. I'll take a look...

Mind approving meanwhile @dberenbaum ? 🙂 thanks

Reply:
Mentioned in https://github.com/iterative/dvc.org/issues/3635 for now.

Reply:
p.s. after https://github.com/iterative/dvc.org/pull/3459 this may not be relevant anyway...